app/src/Liquidator.ts

```diff
@@ -63,7 +81,7 @@ export default class Liquidator {
     this.web3.eth.handleRevert = true;
     this.liquidatorContract = new this.web3.eth.Contract(
       LiquidatorABIJson as AbiItem[],
-      liquidatorAddress,
+      liquidatorAddress
```

Comment:
I thought you preferred trailing commas?

Reply:
My silly linter felt the need to remove it.